Documentation/v2/metrics.md

```diff
@@ -52,7 +24,7 @@ All these metrics are prefixed with `etcd_http_`
 |--------------------------------|-----------------------------------------------------------------------------------------|--------------------|
 | received_total                 | Total number of events after parsing and auth.                                      | Counter(method)        |
 | failed_total                   | Total number of failed events.                                                      | Counter(method,error)  |
-| successful_duration_second     |  Bucketed handling times of the requests, including raft rounds for writes.          | Histogram(method)      |
+| successful_duration_seconds     |  Bucketed handling times of the requests, including raft rounds for writes.          | Histogram(method)      |
```

Comment:
I'd suggest adding in a all requests one then, so the more useful stat is available without breaking existing users.